biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -895,7 +928,7 @@ subroutine UpdatePlantHydrLenVol(ccohort,site_hydr)
     ! alternative cross section calculation
     ! a_sapwood    = a_leaf_tot / ( 0.001_r8 + 0.025_r8 * ccohort%hite ) * 1.e-4_r8
 
-    call CrownDepth(ccohort%hite,crown_depth)
```

Comment:
Why remove the call to crown depth here?

Reply:
@JunyanDing , could we integrate your changes into the CrownDepth subroutine?
https://github.com/NGEET/fates/blob/master/biogeochem/FatesAllometryMod.F90#L1894-L1914

Reply:
@rgknox  yes, do you want me to change that in this pull request or do that after this PR is merged?